scalameter-core/src/main/scala/org/scalameter/keys.scala

```diff
@@ -4,10 +4,12 @@ package org.scalameter
 
 import java.util.Date
 import scala.collection._
+import org.scalameter.picklers.Pickler
+import org.scalameter.picklers.Implicits._
 
 
 
-class Key[T](val name: String)(implicit container: KeyContainer) extends Serializable {
+class Key[T: Pickler](val name: String)(implicit @transient container: KeyContainer) extends Serializable with PicklerBasedKey[T] {
```

Comment:
Besides, what happens after deserialization when one calls `key` and `container` is set to `null`.